src/ansys/aedt/core/application/analysis.py

```diff
@@ -1269,13 +1094,17 @@ def get_nominal_variation(self, with_values=False):
 
         Returns
         -------
-        list of str
-            List of nominal variations.
+        dict
```

Comment:
@Samuelopez-ansys I'd add here an explanation of what is inside dictionary, what is the key and value/values.